src/components/organisms/ChatSidebar/components/PrivateChats/PrivateChats.tsx

```diff
@@ -17,10 +22,14 @@ export interface PrivateChatsProps {
 
 export const PrivateChats: React.FC<PrivateChatsProps> = ({ recipientId }) => {
   const [userSearchQuery, setUserSearchQuery] = useState("");
-  const onInputChange = useCallback(
-    (e) => setUserSearchQuery(e.target.value),
-    []
-  );
+  const [scrollPageNumber, setScrollPageNumber] = useState(1);
+
+  const onInputChange = useCallback((e) => {
+    setUserSearchQuery(e.target.value);
+    setScrollPageNumber(1);
+  }, []);
+
+  const showNextPage = () => setScrollPageNumber(scrollPageNumber + 1);
```

Comment:
use `useCallback`

Reply:
Why? There's no computation done to create the function. I can't see how `useCallback` is going to add value but I may be missing something

Reply:
https://kentcdodds.com/blog/usememo-and-usecallback

Reply:
>  There's no computation done to create the function

Yes there is. If you haven't written it, it doesn't mean the JS engine itself doesn't create function anew having new reference thus possibly having older and newer renders with their own different versions of the handler.

Granted, this is a trivial case and one can go without it, but as a good practice / following established ones in the project, it will be nice to have it in a `useCallback`.

Reply:
As per the @goran-peoski-work 's comment and @dmitrymarchuk note, wrapping the function into `usecallback` ensures that the component doesn't rerender every time when the component is called.

Reply:
https://github.com/sparkletown/sparkle/pull/2057